### DISCUSSION TITLE
Recommend adding a project website url and description

### DISCUSSION THREAD

**Discussion by nezda** [lum-ai/odinson#295] [Ideas]
Recommend adding a project website url and description for the benefit of people who stumble upon your work. Currently:

![image](https://user-images.githubusercontent.com/179091/120446979-b5829a00-c34f-11eb-8c57-c4d73bee2b45.png)

versus say

![image](https://user-images.githubusercontent.com/179091/120447349-23c75c80-c350-11eb-9dd2-fd711850fe0a.png)

**Comment by BeckySharp** [comment]
Done! Thanks for the idea @nezda !
@marcovzla @myedibleenso @danebell @MihaiSurdeanu @kevinlum @kwalcock feel free to modify -- seemed to not be a PR-based process so I exerted my supreme authority 
(you're welcome) :)
